package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -59,13 +66,63 @@ const styles = StyleSheet.create({
     marginLeft: 8,
     marginRight: 8,
   },
+  unsupportedFile: {
+    flexDirection: 'row',
+    paddingLeft: 10,
+  },
+  unsupportedFileText: {
+    fontSize: 13,
+  },
+  warningIconStyle: {
+    borderRadius: 24,
+    bottom: 8,
+    left: 4,
+    right: 0,
+    top: 2,
+  },
 });
 
+const UnsupportedFileTypeIndicator = ({
+  indicatorType,
+  item,
+}: {
+  indicatorType: ProgressIndicatorState;
+  item: FileUpload;
+}) => {
+  const {
+    theme: {
+      colors: { accent_red, grey },
+      messageInput: {
+        fileUploadPreview: { fileSizeText },
+      },
+    },
+  } = useTheme();
+
+  const { t } = useTranslationContext();
+  return (indicatorType as unknown as string) === ProgressIndicatorTypes.NOT_SUPPORTED ? (
```

Comment:
` as unknown as string` probably means there's something wonky with the typing of `indicatorType` - this makes the type unnecessary. Can you remove it, and rather figure out what the root cause of this is, so the type of it is correct?